packages/frontend/src/components/UserSettings/AllowedDomainsPanel/index.tsx

```diff
@@ -1,161 +1,186 @@
-                onSubmit={handleUpdate}
-                disableSubmitOnEnter
-                isLoading={isLoading}
-            >
-                <TagInput
-                    label="Allowed email domains"
+    return isSuccess ? (
+        <form name="allowedEmailDomains" onSubmit={handleOnSubmit}>
+            <Stack>
+                <MultiSelect
                     name="emailDomains"
+                    label="Allowed email domains"
                     placeholder="E.g. lightdash.com"
                     disabled={isLoading}
-                    defaultValue={data?.emailDomains || []}
-                    rules={{
-                        validate: {
-                            isValidEmailDomain:
-                                isValidEmailDomain('Email domains'),
-                            isValidOrganizationDomain:
-                                isValidOrganizationDomain('Email domains'),
-                        },
+                    data={form.values.emailDomains.map((emailDomain) => ({
+                        value: emailDomain,
+                        label: emailDomain,
+                    }))}
+                    searchable
+                    creatable
+                    getCreateLabel={(query: string) => `+ Add ${query} domain`}
+                    defaultValue={form.values.emailDomains}
+                    onCreate={(value) => {
+                        if (!isValidEmailDomain(value)) {
+                            form.setFieldError(
+                                'emailDomains',
+                                `${value} should not contain @, eg: (gmail.com)`,
+                            );
+                            return;
+                        }
+
+                        const isInvalidOrganizationEmailDomainMessage =
+                            validateOrganizationEmailDomains([
+                                ...form.values.emailDomains,
+                                value,
+                            ]);
+                        if (isInvalidOrganizationEmailDomainMessage) {
+                            form.setFieldError(
+                                'emailDomains',
+                                isInvalidOrganizationEmailDomainMessage,
+                            );
+                            return;
+                        }
+
+                        return value;
                     }}
+                    {...form.getInputProps('emailDomains')}
                 />
-                {selectedEmailDomains.length > 0 && (
+
+                {!!form.values.emailDomains.length && (
                     <>
-                        <Select2
+                        <Select
                             label="Default role"
                             name="role"
                             placeholder="Organization viewer"
                             disabled={isLoading}
-                            items={roleOptions}
+                            data={roleOptions}
```

Comment:
Something is wrong with this component. Has duplicated options. I was able to select the same project multiple times. 
Would be nice to see what projects are selected or not in the dropdown.


![Captura de ecrã 2023-05-17, às 16 25 48](https://github.com/lightdash/lightdash/assets/9117144/79adfc8b-bb4c-4a7a-a104-b12b19f79570)

Reply:
I think there's an issue with the Render-deployed env, look at the project list dropdown on the nav bar, also has duplicates 🤔  
![image](https://github.com/lightdash/lightdash/assets/7611706/8126f482-7eaf-4e84-aac5-3a060cdac5b4)